### DISCUSSION TITLE
Truecharts

### DISCUSSION THREAD

**Discussion by KickingAss2024** [karlomikus/bar-assistant#294] [Ideas]
I am trying to get your application on truecharts that allowed use to use bas assessment with kubernetes. 

I have started a PR, would love some help with it. 

https://github.com/truecharts/charts/pull/22741

**Comment by karlomikus** [comment]
I've never worked with Helm/Kubernets so unfortunately I can't help you.

**Comment by zhdenny** [comment]
Kubernates and Docker are two very different things.

TrueNas Scale has already mentioned they plan to support Docker in the future.

But honestly, if I were you I would give serious thought to NOT using Kubernates/TrueCharts....instead, fire up a Linux VM in TrueNas, install docker on the VM and be on your way.

I'm sure you've heard this advice be given before by other TrueNas Scale users.....

It will make your life so much easier since majority of services support docker first, and maybe Kubernates TrueCharts stuff later but it's an afterthought.

**Reply by KickingAss2024** [reply]
I appreciate your advice, but it seems that most of the information you've provided suggests that I should discontinue using a project that is already up and running.

It would be the same as I said to others not to use this project..... 

I recommend considering hosting this open-source project on an additional platform to make it accessible to a wider audience. (docker and kubernetes) 

As soon as this feature becomes available on Truechats, I'll share it here.

There's no need to debate the optimal platform for running Bar Assistant.
